kivy/uix/actionbar.py

```diff
@@ -307,7 +307,7 @@ def _toggle_dropdown(self, *largs):
         children = ddn.container.children
 
         if children:
-            ddn.width = max([self.width, children[0].minimum_width])
+            ddn.width = max(self.width, max(c.minimum_width for c in children))
```

Comment:
In other circumstances I would, but in this case: nothing can be scheduled during iteration, so children cannot be modified (it is not legal to do it from a different thread), and we may as well not incur the overhead of a copy. or something I missed?